tests/ocamlgen_test.ml

```diff
@@ -1,5 +1,44 @@
-(** This file is used to test executables.
-    It simply links the library so that code there is executed and tested.
-  *)
+let test_single_tuple_access_field () =
+  let b = Bindings.new_t () in
+  assert (b.inner = "Hello") ;
+  (* COMMENTME: what do we test here? *)
```

Comment:
Yes indeed. What do we test here?

Reply:
:D I am in favor of removing this old code as it does not seem to be testing anything. See fixup 89fbfb7d176d790d95844f6fe0ce337c5d1a01a9.